### PR TITLE
adding language support for portuguese i18n

### PR DESCRIPTION
**Pull request by ninalencastro**
Includes Portuguese to the list of languages available for testing. Translation is complete and awaiting review.

### CODE REVIEW

**Review by mekarpeles — APPROVED** [review]
looks good to me, thanks for your contributions!

**Comment by ninalencastro** [comment]
So, I have almost zero knowledge of python, so idk what part of the coding is off, but if anyone with knowledge of it would like to take a look, feel free too! Unfortunately my ability to alter this file extends itself only to the translation.

**Comment by cdrini** [comment]
Thank you so much @ninalencastro ! Portuguese is the third most used language by Open Library users, so this is going to be great! Let me take a look at what's happening here...

File: openlibrary/i18n/pt/messages.po [hunk 1]
```diff
@@ -0,0 +1,5280 @@
...
+# Copyright (C) 2018 Internet Archive
+# This file is distributed under the same license as the Open Library
+# project.
+# FIRST AUTHOR <NINA.ALENCASTRO@GMAIL.COM>, 2022.
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: Open Library VERSION\n"
+"Report-Msgid-Bugs-To: NINA.ALENCASTRO@GMAIL.COM\n"
+"POT-Creation-Date: 202-11-23 08:44+0000\n"
```
Comment (cdrini) [inline openlibrary/i18n/pt/messages.po:11]:
```suggestion
"POT-Creation-Date: 2022-11-23 08:44+0000\n"
```

Ah, there was a typo in the date!

File: openlibrary/i18n/pt/messages.po [hunk 2]
```diff
@@ -0,0 +1,5280 @@
...
+# Copyright (C) 2018 Internet Archive
+# This file is distributed under the same license as the Open Library
+# project.
+# FIRST AUTHOR <NINA.ALENCASTRO@GMAIL.COM>, 2022.
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: Open Library VERSION\n"
+"Report-Msgid-Bugs-To: NINA.ALENCASTRO@GMAIL.COM\n"
+"POT-Creation-Date: 2022-11-23 08:44+0000\n"
```
Comment (cdrini) [inline openlibrary/i18n/pt/messages.po:11]: Oops! Wrong year
```suggestion
"POT-Creation-Date: 2022-08-08 08:44+0000\n"
```

File: openlibrary/i18n/pt/messages.po [hunk 3]
```diff
@@ -0,0 +1,5280 @@
...
+msgstr "páginas"
+
+#: type/edition/view.html:230 type/work/view.html:230
+#, python-format
+msgid ""
+"This edition doesn't have a description yet. Can you <b><a "
+"href='%(url)s'>add one</a></b>?"
+msgstr ""
+"Esta edição ainda não tem uma descrição. Você poderia <b><a "
+"href='%(url)s'adicionar uma</a></b>?"
```
Comment (cdrini) [inline openlibrary/i18n/pt/messages.po:4372]: Fix missing html tag
```suggestion
"href='%(url)s'>adicionar uma</a></b>?"
```

**Comment by cdrini** [comment]
Merged! Can you also add Portuguese to the list here? That'll make it appear as an option in the UI :)

**Comment by ninalencastro** [comment]
> Merged! Can you also add Portuguese to the list here? That'll make it appear as an option in the UI :) https://github.com/internetarchive/openlibrary/blob/3d0c903f7847aac9b61cd422002652a8175bd79f/openlibrary/templates/languages/language_list.html

Sure! How do I do that? Change it on my fork or...? I'm not used to how GitHub works.

**Comment by cdrini** [comment]
Yep! Exactly. If you're using the terminal, these instructions show what you need to do: https://github.com/internetarchive/openlibrary/wiki/Git-Cheat-Sheet#creating-a-pull-request

If you're more comfortable using the github UI, press the pencil button in the top right: https://github.com/internetarchive/openlibrary/blob/master/openlibrary/templates/languages/language_list.html , and select "Create new branch" at the bottom of the page after making your edit!

**Pull request merged**